gdx/src/com/badlogic/gdx/Input.java

```diff
@@ -570,6 +570,12 @@ private static void initializeKeyNames () {
 	/** @return The value of the accelerometer on its y-axis. ranges between [-10,10]. */
 	public float getAccelerometerZ ();
 
+	public float getGyroscopeX ();

```

Comment:
Should have some javadoc as to what is expected to return from these functions, namely the un-normalized gyro sample data.

"return The rate of rotation around the x axis. (rad/s)" should be good enough.